docs/release/upgrade.adoc

```diff
@@ -45,6 +45,30 @@ When upgrading to new feature versions of Enonic XP, new versions of standard ap
 After upgrading XP, check for new versions of your installed applications from Enonic market.
 We generally always recommend updating your apps to the latest version!
 
+== v7.15 notes
+
+* The image service now generates progressive JPEG images by default. This can be changed back in <<deployment/config#image>> by setting `progressiveJpeg` to empty string.
+
+* GZIP HTTP compression is disabled by default. It can be enabled in <<deployment/config#jetty, Jetty config>>. By setting `gzip.enabled = true`.
+
+* When content is created `image/webp` and `image/avif` media types create `media:image` content type, instead of `media:unknown`. Old versions of Content Studio might not work correctly with WebP and AVIF media types. Upgrae t the latest version of Content Studio.
```

Comment:
```suggestion
* `image/webp` and `image/avif` media types will now produce content of `media:image` type, instead of `media:unknown`. Old versions of Content Studio might not work correctly with WebP and AVIF media types. Upgrade to the latest version of Content Studio.
```